INN/umbrella-currentorg
# Internal qa updates

## Changes

This pull request makes the following changes:

- Makes `0.5rem` margin be on all tags on the list holder items instead of all but the first and switches it from `margin-left` to `margin-right` so it's inline with the content
- Adds 6px left and right padding to project statuses on the single view
- Adds 1rem bottom margin to images and videos
![Screen Shot 2020-06-09 at 2 01 03 PM](https://user-images.githubusercontent.com/18353636/84183304-b133e180-aa59-11ea-8e25-61749acc35e0.png)
- Makes expanded category holder 175px wide instead of 100% so it doesn't cover everything
![Screen Shot 2020-06-09 at 1 54 38 PM](https://user-images.githubusercontent.com/18353636/84182702-cbb98b00-aa58-11ea-85ba-059afefc5fc2.png)
- Adds support for `excerpt` in projects custom post type


## Why



For #143

## Testing/Questions

Features that this PR affects:

- Tags/categories on the list items
- Images and videos
- Category filter
- Project excerpt field


Questions that need to be answered before merging:


Steps to test this PR:

1. Poke around and make sure everything looks ok
2. Make sure the excerpt field is visible in the project post editor
